eventsapi/models/api.py

```diff
@@ -1,7 +1,10 @@
 from typing import Annotated, Literal, Union, Optional
-from pydantic import Field, BaseModel
+from pydantic import Field, BaseModel, AnyHttpUrl
 from uuid import UUID
 
+CONTENT_LOADED_V1 = 'content_loaded_v1'
+CONTENT_LOAD_FAILED_V1 = 'content_load_failed_v1'
```

Comment:
I think you previously removed all references to these but left the constants. We can delete these lines now.